<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LahjoitaRahaaWindow</class>
 <widget class="QDialog" name="LahjoitaRahaaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lahjoitus</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lahjoita Rahaa</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="mainCharitiesLayout">
       <item>
        <layout class="QVBoxLayout" name="leftCharitiesLayout">
         <item>
          <widget class="QPushButton" name="pelastakaaLapsetButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Pelastakaa Lapset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="punainenRistiButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Punainen Risti</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pelastusarmeijaButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Pelastusarmeija</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="rightCharitiesLayout">
         <item>
          <widget class="QPushButton" name="syopasaatioButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Syöpäsäätiö</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mieliRYButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Mieli RY</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="takaisinButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>TAKAISIN</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
